core-bundle/src/Image/Studio/FigureBuilder.php

```diff
@@ -705,6 +710,11 @@ private function validatorAdapter()
         return $framework->getAdapter(Validator::class);
     }
 
+    private function eventDispatcher()
```

Comment:
```suggestion
    private function eventDispatcher(): EventDispatcherInterface
```

Should we use the `Symfony\Contracts\EventDispatcher\EventDispatcherInterface` here?

Reply:
sounds good